docs/articles/roadmap/README.md

```diff
@@ -15,28 +16,30 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **In Progress**
-[API Testing](https://github.com/DevExpress/testcafe/issues/4288)                                 | Planned
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
```

Comment:
Should we merge https://github.com/DevExpress/testcafe/issues/2397 with https://github.com/DevExpress/testcafe/issues/4288? I think we don't need them both

Reply:
I closed #2397.